src/v2/Apps/FairOrginizer/FairOrganizerApp.tsx

```diff
@@ -14,10 +15,20 @@ interface FairOrganizerAppProps {
 const FairOrganizerApp: React.FC<FairOrganizerAppProps> = ({
   fairOrganizer,
 }) => {
-  const { name } = fairOrganizer
+  const { name, profile, slug } = fairOrganizer
+
+  const title = `${name} | Artsy`
+
   return (
     <>
-      <Title>{`${name} | Artsy`}</Title>
+      <MetaTags
+        description={`Search for fairs on ${name} on Artsy`}
+        imageURL={profile?.image?.url}
+        pathname={`fair-organizer/${slug}`}
+        title={title}
+      />
+
+      <Title>{`${title}`}</Title>
```

Comment:
We can get rid of this! `<title>` is [included in MetaTags](https://github.com/artsy/force/blob/3f64ac9d19cd841d5147f18653540124d3b385d1/src/v2/Components/MetaTags.tsx#L51)